### PR TITLE
Optimize  bundleLogger.js

### PR DESCRIPTION
**Pull request by Shyam-Chen**
Optimize?!

### CODE REVIEW

File: gulp/util/bundleLogger.js [hunk 1]
```diff
@@ -8,19 +8,21 @@
...
-var startTime;
-
 export default {
 
   start() {
-    startTime = process.hrtime();
-    gutil.log('Running', gutil.colors.green('\'bundle\'') + '...');
+    let startTime = process.hrtime();
+    gutil.log('Starting', `
+      '${gutil.colors.cyan('bundle')}'...
```
Comment (jakemmarsh) [inline gulp/util/bundleLogger.js]: could you put these `gutil.log` params on the same line? They don't look long enough to need wrapping

File: gulp/util/bundleLogger.js [hunk 2]
```diff
@@ -8,19 +8,21 @@
...
   },
 
   end() {
-    var taskTime = process.hrtime(startTime);
-    var prettyTime = prettyHrtime(taskTime);
-    gutil.log('Finished', gutil.colors.green('\'bundle\''), 'in', gutil.colors.magenta(prettyTime));
+    let taskTime = process.hrtime(startTime);
+    let prettyTime = prettyHrtime(taskTime);
+    gutil.log('Finished', `
+      '${gutil.colors.cyan('bundle')}' after ${gutil.colors.magenta(prettyTime)}
```
Comment (jakemmarsh) [inline gulp/util/bundleLogger.js]: could you put these `gutil.log` params on the same line? They don't look long enough to need wrapping

**Comment by Shyam-Chen** [comment]
```coffeescript
# coffeescript
class TimeLogger

  start = null

  startFn: (task) ->
    start = process.hrtime()
    gutil.log 'Starting', "'#{gutil.colors.cyan task}'..."

  endFn: (task) ->
    end = process.hrtime start
    words = prettyHrtime end
    gutil.log 'Finished', "
      '#{gutil.colors.cyan task}' after #{gutil.colors.magenta words}
    "

timeLogger = new TimeLogger()
```

**Comment by Shyam-Chen** [comment]
https://github.com/Shyam-Chen/angular-starter-kit/blob/master/gulpfile.coffee#L77

**Pull request closed without merging**